build/UserALEWebExtension/background.js

```diff
@@ -428,6 +428,30 @@ var intervalCounter;
 var intervalLog;
 var cbHandlers = {};
 
+/**
+ * Adds named callbacks to be executed when logging.
+ * @param  {Object } newCallbacks An object containing named callback functions.
+ */
+function addCallbacks() {
```

Comment:
Please add some comments to this function to help describe what's happening here at a business logic level.